SocialFilter/sections/specific.txt

```diff
@@ -48,6 +48,7 @@ japanese.joins.com,sedaily.com,canshop.jp,koreatimes.co.kr,m.chosun.com,sports.d
 ! ##.content__post--meta-social
 otomobilgunluklerim.com,tilki.net,ekosayfa.com,bilisimogretmeni.com,azbuz.org,siberkalem.com,teknobird.com,buneymis.net,teknoxhaber.com,fosforludusunceler.com,kirtkirtla.com,oyuncukuru.org,denizcilikbilgileri.com,hknkr.com,kadioglublog.com##.content__post--meta-social
 !
+ototoy.jp##.social-button
```

Comment:
You'll also block this. Will fix it.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/147731290-186383f0-bbf0-4673-b6ca-92d863c500cf.png)

</details><br/>